### ISSUE TITLE
Turn On/Off Display on Samsung AC

### ISSUE THREAD

**Issue by olympia** [atxbyea/samsungrac#30]
Would it be possible to add State attribute for the Display on Samsung AC and have it turned ON/Off by Home Assistant automation?

**Comment by atxbyea** [comment]
Does your model support that ? Mine does not.

**Comment by olympia** [comment]
Yes, not exactly the same remote (I have a dedicated settings button where the "Display" setting is located", but it works same as this one: https://www.youtube.com/watch?v=Rhq16dQJNjA

Is there any way I can contribute in this?

**Comment by wifi75** [comment]
it would be very interesting to turn off the display ...
